script/install.sh

```diff
@@ -421,6 +421,17 @@ resolve_latest() {
             logV "Determining latest ldc-beta version ($url)."
             COMPILER="ldc-$(fetch $url)"
             ;;
+        ldc-nightly)
```

Comment:
I'd say we should at least also resolve `ldc-2017222` so it's possible to install a compiler from a specific date, quite useful for bisecting an issue.
I'm not sure whether you actually plan to archive your nightly builds.

Reply:
I also don't know whether we'll archive the nightlies.
This is a new feature though, so please just merge this.

Reply:
(they aren't "nightlies" btw, it's a new build after each git commit)

Reply:
Perhaps we should name it `ldc-latest-ci-build`?

Reply:
I don't like the term 'nightly' either. I guess for DMD they are indeed proper nightlies with no difference compared to regular builds. For us, they are the latest successful CI builds, which means a revision may not be available for all platforms (sporadic failures, timeouts...), not available at the same time (Windows x64/multilib and OSX packages taking quite a bit longer than the Linux x64 one), and particularly, they feature enabled LLVM+LDC assertions (well, currently not for Windows, due to AppVeyor slowness), increasing compile times.